styles.css

```diff
@@ -76,3 +79,175 @@ a {
   width: 22px;
   padding-right: 15px;
 }
+
+#projects {
+  background-color: #f7f7f9;
+  padding-top: 100px;
+  padding-bottom: 30px;
+}
+
+.project {
+  background-color: #fff;
+  margin-top: 30px;
+  margin-right: 24px;
+  margin-left: 24px;
+  border-radius: 16px 16px 16px 16px;
+  border: 1px solid #dfe1e6;
+}
+
+.project-image {
+  display: flex;
+  justify-content: center;
+  padding-left: 15px;
+  padding-right: 15px;
+  padding-top: 15px;
+}
+
+.project-content {
+  padding-left: 15px;
+  padding-right: 15px;
+}
+
+.project-heading {
+  color: #172b4d;
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 44px;
+  margin-bottom: 10px;
+  margin-top: 10px;
+}
+
+.content-text {
+  font-size: 13px;
+  font-weight: 600;
+  line-height: 16px;
+  color: #7a869a;
+  display: inline;
+}
+
+.canopy {
+  color: #344563;
+}
+
+.count-vector {
+  padding-left: 5px;
+  padding-right: 5px;
+}
+
+.description-text {
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 24px;
+  padding-right: 19px;
+  color: #344563;
+}
+
+.language-used {
+  background-color: #ebebff;
+  color: #6070ff;
+  padding: 4px 12px 4px 12px;
+  font-size: 12px;
+  border-radius: 8px 8px 8px 8px;
+}
+
+.project-link {
+  color: #396df2;
+  font-size: 17px;
+  font-weight: 500;
+  line-height: 24px;
+}
```

Comment:
Good job for the design of the button 💯 

However, please add the hover effect like asked in the requirement

<img width="471" alt="image" src="https://user-images.githubusercontent.com/7243132/170837677-1a225383-2336-4bf9-b192-c4faab446c26.png">

<img width="468" alt="image" src="https://user-images.githubusercontent.com/7243132/170837702-9740f86f-a800-4b3b-9b09-2b9b351a0b74.png">